src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <v-app dark :style="cssVars">
+    <v-app :style="cssVars">
```

Comment:
I think this will be wrong, if you just delete the dark option with no condition.